docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -19,31 +31,54 @@ Then, to configure your network you just need to follow these three steps:
 ```yaml
 Settings:
     APN: virtueyes.com.br # Access Point Name
+    Module: BG95M3 # Module model (BG770A, BG95M3 or M95)
     User: virtu # APN user (optional)
     Password: virtu # APN password (optional)
     Operator: 72410 # Carrier numeric operator code (optional)
-    Mode: 7 # Network mode (Cat-M1 = 7, Nb-IoT = 9)
+    Mode: CATM1 # Network mode (CATM1, NBIOT or GSM) (optional)
 ```
 
-***Notes**: It's recommended to define the carrier operator code, since it avoids the module network automatic selection, making the connection generally more reliable.*
+> **Notes**: If the carrier numeric operator code (**Operator**) or the network mode is not specified (**Mode**), the module will attempt to automatically determine the optimal network based on the M2M sim card inserted and your location. However, if you encounter any connectivity issues, it is advisable to manually configure these settings.
 
-2 - Select **BG770A** as **DefaultInterface** in the **meadow.config.yaml**, if you don't have this *yaml* file in your device, you should create and flash it to the device:
+2 - Select `Cell` as `DefaultInterface` in the **meadow.config.yaml**, if you don't have this *yaml* file in your device, you should create and flash it to the device:
 
 ```yaml
 Network:
-    DefaultInterface: BG770A
+    DefaultInterface: Cell
 ```
 
-3 - Reserve the pins **I9**, **H13** and **C7**, which are used to turn on the **BG770A Cell Wing** and to exchange data with the **F7 Feather v2**:
+3 - Reserve the pins `I9` and `H13`, which are used to exchange data between the modem and the **Meadow F7v2 Feather**, and the `C7`, which is used to turn on the modules:
 
 ```yaml
 Device:
     ReservedPins: I9;H13;C7
 ```
 
+> **Notes**: You can find the pins `I9`, `H13` and `C7` as `D00`, `D01` and `D10` on the **Meadow F7v2 Feather**, respectively.
+
 ## Hardware configuration
 
-To use the **BG770A Cell Wing with the F7 Feather v2**, you just need to attach them, and connect an *LTE* antenna to the cell module.
+To set up the hardware, the easiest way is to use our **BG770A Cell Wing** by attaching it to the **Meadow F7v2 Feather**. This provides a straightforward and hassle-free solution for establishing cellular connectivity. However, you can also use other cell click board models. All you need to do is connect the click boards to the **Meadow F7v2 Feather** using a few jumpers. To set up the hardware, follow the instructions based on your cell click board model:
+
+### Quectel BG770A - Meadow Cell Wing
+To utilize the **BG770A Cell Wing** with the **Meadow F7v2 Feather**, simply attach them, connect an LTE antenna (Rubber ducky or Dome) with an SMA Plug connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert an **M2M** SIM card into the cell module.
+
+### Quectel BG95-M3 - NimbeLink Skywire click board
```

Comment:
I'm specifying the click board that we are using since there might be other click boards for the same module with different power-up requisites and/or different pin nomenclatures. What do you think about that? @alancassis @alexischr